packages/core/src/components/Search/Search.tsx

```diff
@@ -116,8 +118,10 @@ const Search = forwardRef(
         inputRole={searchResultsContainerId ? "combobox" : undefined}
         aria-label={inputAriaLabel}
         aria-busy={loading}
-        aria-owns={searchResultsContainerId}
+        aria-controls={ariaExpanded ? searchResultsContainerId : undefined}
```

Comment:
in addition, after another read on the differences (wasn't aware they are related), it sounds like `controls` mean that it can be controlled through other element, e.g. if I choose something from some other component, it would change the Search input content

with `owns`, we said that it is contained in another element, even though they're not in the same DOM hierarchy.